release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1247,6 +1247,10 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
 
 * With Serverless 0.10, the latest supported versions were updated for `trigger`, `subscription`, `channel`, and `IMC`.  Static models corresponding to each showed an API version of `beta`. The API version for eventing resources is now updated to `v1` and the UI now shows the latest supported version. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1890104[*BZ#1890104*])
 
+* Previously, when the user switched between workloads on the *Monitoring Dashboard* tab, for example from deployment to *All workloads*, the dashboard displayed a white canvas and no chart was displayed. This issue has been fixed now and the dashboard displays charts when the user switches between workloads. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1911129[*BZ#1911129*])
+
+* Previously, the *Monitoring Alert* icon was missing on the workload in the *Topology* view. Currently, the following severity labels are being supported: `critical`, `warning`, `info` and `none`. This issue has been fixed now and anything apart from the above mentioned labels is treated as `warning`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925200[*BZ#1925200*])
```

Comment:
The Monitoring alert decorator didn't show up because anything apart from the mentioned labels was treated as `info` and we do not show info decorator on the topology node.

How about if we can phrase it like:
```suggestion
* Previously, anything apart from `critical`, `warning`, `info` and `none` was treated as `info`. As a result, the *Monitoring Alert* decorator was not displayed on the workload in the *Topology* view for a severity label like `Critical`. This issue has been fixed now and anything apart from the above mentioned labels is treated as `warning`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925200[*BZ#1925200*])
```

Reply:
Thanks @debsmita1

Fixed!